cli/bench/deno_http.js

```diff
@@ -0,0 +1,68 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+async function main(addr) {
+  const rid = await Deno.http.createServer(addr);
+  console.log("Server listening on", addr);
+
+  while (true) {
+    const connRid = await Deno.http.accept(rid);
+    handleConn(connRid);
+  }
+}
+
+async function handleConn(connRid) {
+  while (true) {
+    const [connectionClosed, ...req] = await Deno.http.nextRequest(connRid);
```

Comment:
should avoid creating new array and using iterator protocol here

Reply:
Not sure what you're suggesting here, care to explain?

Reply:
Removed destructuring altogether